controllers/chat/getChannelDetail.js

```diff
@@ -36,13 +36,13 @@ const getChannelDetail = async (req, res) => {
     });
   }
 
-  const {betterChannelMember, betterChannelMemberObject} =
+  const {betterChannelMember, betterChannelMemberObject, updatedChannel} =
     await BetterSocialCore.chat.updateBetterChannelMembers(channel, createdChannel, true);
 
   return ResponseSuccess(res, 'Success', 200, {
     better_channel_members: betterChannelMember,
     better_channel_member_objects: betterChannelMemberObject,
-    ...createdChannel,
+    ...updatedChannel,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here seem to be a simple renaming of the variable `createdChannel` to `updatedChannel`. This change is likely for better readability and understanding of the code. However, it's important to ensure that this new variable `updatedChannel` is correctly defined and updated in the `updateBetterChannelMembers` function from `BetterSocialCore.chat`. If not, this could lead to undefined behavior or errors.
```diff
-    ...createdChannel,
+    ...updatedChannel,
```
If `updatedChannel` is indeed correctly defined and updated in the `updateBetterChannelMembers` function, then these changes look good to me.


<!-- This is an auto-generated comment by OSS CodeRabbit -->